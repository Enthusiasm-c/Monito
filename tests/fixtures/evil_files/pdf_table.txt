
PDF TABLE SIMULATION
====================

ПРАЙС-ЛИСТ ТОВАРОВ 2024
Компания: ООО "Тест Системс"
Дата обновления: 15.01.2024

┌─────────────────────┬──────────┬─────────────┬──────────────┐
│ Название товара     │ Цена     │ Категория   │ Примечание   │
├─────────────────────┼──────────┼─────────────┼──────────────┤
│ iPhone 15 Pro       │ 120,000  │ Смартфоны   │ Новинка 2024 │
│ MacBook Air M2      │ 150,000  │ Ноутбуки    │ В наличии    │
│ iPad Pro 12.9"      │ 80,000   │ Планшеты    │ Под заказ    │
│ AirPods Pro 2       │ 25,000   │ Аксессуары  │ Скидка 10%   │
│ Apple Watch Ultra   │ 95,000   │ Часы        │ Водонепр.    │
└─────────────────────┴──────────┴─────────────┴──────────────┘

Итого товаров: 5
Общая стоимость: 470,000 руб.

Условия доставки:
- Москва: бесплатно от 50,000 руб.
- Регионы: 1,500 руб.

Контакты: sales@testsystems.ru
Телефон: +7 (495) 123-45-67
        