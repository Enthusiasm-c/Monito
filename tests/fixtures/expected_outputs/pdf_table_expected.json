{
  "filename": "pdf_table.txt",
  "expected_rows_in": 5,
  "expected_rows_out": 5,
  "format": "mock_pdf",
  "challenges": [
    "PDF table extraction",
    "ASCII table parsing",
    "Non-standard format",
    "Mixed content"
  ]
}